app/org/maproulette/framework/mixins/TaskFilterMixin.scala

```diff
@@ -75,22 +76,12 @@ trait TaskFilterMixin {
         query.addFilterGroup(
           FilterGroup(
             List(
-              BaseParameter(
-                "id",
-                None,
-                Operator.NULL,
-                useValueDirectly = true,
-                table = Some("l")
-              ),
-              BaseParameter(
-                "user_id",
-                user.id,
-                Operator.EQ,
-                useValueDirectly = true,
-                table = Some("l")
+              CustomParameter(
```

Comment:
This query seems like it would be completely different, not sure if I am just reading it incorrectly.